all-of-us/workbench
# [RW-4462][risk=low] Add GCS traffic and hook up delete-cluster button to WS admin page.

The main addition in this PR is to pipe cloud storage metrics into our Workspace admin page. I also made a handful of drive-by changes to the styling, and hooked up the button to delete a cluster. This gets us 95% of the way there to be able to respond quickly to an egress event. (The remaining piece is probably to change the routing to create a per-workspace page.)

I manually tested a full cluster deletion cycle:
 * Open a second tab with a notebook running, copying a large file to the bucket.
 * Refresh the ws-admin page, see the spike in GCS traffic.
 * (In the real-world, that signal would indicate that we should *not* shut down the cluster.)
 * To test the delete-cluster, I clicked "Delete" on the active cluster.
 * My notebook in the other tab stopped working after around ~10 seconds.

Screenshots:

<img width="100%" alt="cloud-storage-traffic" src="https://user-images.githubusercontent.com/51842/75352460-c81e4180-5877-11ea-9a20-94ebd607a283.png">

<img width="50%" alt="workspace-details-layout" src="https://user-images.githubusercontent.com/51842/75352466-c9e80500-5877-11ea-9f87-5eaf428b7e34.png">


<img width="50%" alt="delete-cluster-conf" src="https://user-images.githubusercontent.com/51842/75352416-b9378f00-5877-11ea-89b4-302349408f3d.png">


---
**PR checklist**
